hathor/websocket/factory.py

```diff
@@ -197,7 +197,10 @@ def serialize_message_data(self, event, args):
             data = tx.to_json_extended()
             data['is_block'] = tx.is_block
             # needed to check if the transaction is from the mempool
-            data['first_block'] = tx.get_metadata().first_block
```

Comment:
Should we add a unittest in this PR?

Reply:
That would certainly be great. But I invested no time into this. Should I do this?

Reply:
@pedroferreira1 went ahead and patched a test to cover this regression, I also checked it and it does fail without the fix.